Made getting started example more comprehensive

vellamike opened the pull request:
Signed-off-by: Mike Vella <vellamike@gmail.com>

# Description

The example in the "getting started" guide was not end-to-end - the user needed to rename several variables, move tensors to GPU, add imports etc. This change makes the getting started guide more comprehensive.

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- Documentation enhancement

# Checklist:

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas and hacks
- [ x] I have made corresponding changes to the documentation and have regenerated the documentation (`make html` in docsrc)
- [ N/A] I have added tests to verify my fix or my feature
- [ N/A] New and existing unit tests pass locally with my changes

narendasan commented on docsrc/tutorials/getting_started.rst:
```diff
@@ -138,20 +147,23 @@ to load in a deployment application. In order to load a TensorRT/TorchScript mod
-                "opt": [1, 3, 512, 512],
-                "max": [1, 3, 1024, 1024]
-            }, # For static size [1, 3, 224, 224]
+                "min": [1, 1, 32, 32],
```
Lets vary the sizes here, it is supposed to demonstrate dynamic shape

vellamike replied:
done

vellamike commented:
@narendasan the ode sample in the documentation will not run until next release where [this commit](https://github.com/NVIDIA/TRTorch/commit/4eb20bbded54f12d220f91bb708f1c11e44aad63) is included. Suggest holding off merging this PR until that release.

narendasan commented:
Seems like old versions of the docs are getting deleted. How did you generate the docs? All you should need to do is `make clean && make html` which will remove the old master branch docs and regenerate them without touching the old versions

vellamike commented:
> Seems like old versions of the docs are getting deleted. How did you generate the docs? All you should need to do is `make clean && make html` which will remove the old master branch docs and regenerate them without touching the old versions

You are right, I am not entirely sure what went wrong, but I reverted that commit and re-did the relevant changes. The issue is now fixed.

narendasan commented:
@vellamike is this PR ready to merge? We are looking to cut a release this week. The PR looks fine to me with the fix for the older documentation.

vellamike commented:
Yes, it's good to merge.

On Thu, 9 Jul 2020, 20:47 Naren Dasan, <notifications@github.com> wrote:

> *@narendasan* approved this pull request.
>
> LGTM
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/NVIDIA/TRTorch/pull/125#pullrequestreview-445918815>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AALYB7OBKGMJ43ONK2C2HA3R2YNEZANCNFSM4OURAJGQ>
> .
>

narendasan commented:
Looks like there was an issue with the DCO signing 

```

You only have one commit incorrectly signed off! To fix, head to your local branch and run:

git commit --amend --signoff

Now your commits will have your sign off. Next run

git push --force-with-lease origin getting-started-guide-improvments

Commit sha: c76208f, Author: Mike Vella, Committer: Mike Vella; Expected "Mike Vella vellamike@gmail.com", but got "Mike Vella vellamike@gmailcom".
```

narendasan commented:
I rebased this branch and fixed the signing issue in PR #135 so closing this PR in favor of that one.
